Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -228,6 +228,24 @@ private void InitializeKeyCommands()
                 }
             });
 
+            InsertSuggestion = new RelayCommand(_ =>
+            {
+                var results = SelectedResults;
+
+                var result = results.SelectedItem?.Result;
```

Comment:
can just combine these two
```suggestion
                var result = SelectedResults.SelectedItem?.Result;
```